scripts/prepare.js

```diff
@@ -37,6 +38,6 @@ if (!existsSync(FILE_HUSKY)) {
 
 // this will create .husky/commit-msg if it does not yet exist.
 if (!existsSync(FILE_COMMIT)) {
-  global.console.log(CLI_COMMIT);
+  logger.info(CLI_COMMIT);
```

Comment:
Do we want to make this change? Surely we want the Winston logger to be used for Git Proxy related services only not necessarily development tooling?